spec/models/item_spec.rb

```diff
@@ -1,6 +1,13 @@
 require "rails_helper"
 
 RSpec.describe Item, type: :model do
+  before (:each) do
+    @merchant = Merchant.create!(name: "Booze Shop")
```

Comment:
I like the descriptive instance variables for more items, better than our singular/generic ones.